docs/end-to-end-demo.md

```diff
@@ -69,7 +69,7 @@ The following will be covered in this demo:
     > ```
 
 ## Setting up a cluster
-Reference our [cluster setup documentation](./setting-up-cluster.md) to set up a cluster or adapt your currently existing cluster. 
+Reference our [cluster setup documentation](./setting-up-cluster.md) to set up a **single-node** cluster or adapt your currently existing cluster. This documentation assumes you are using a single-node cluster, so if installing K3s or MicroK8s, you can skip the last step of the installation instructions.
```

Comment:
Curious about the emphasis? Can we not emphasize that or also mention that you can bring your own cluster with any number of nodes? Otherwise it might sound that BYOC needs to be a single node too.

Reply:
In a way it does need to be single node too. Or at least to follow the documentation exactly.  I want the demo to be as quick as possible to do, so i'd prefer people dont try to set up multinode clusters if they dont already have a cluster

Reply:
Ok, perhaps then add a disclaimer that you can use multi node cluster this demo, but be aware that you might see multiple instances where we show one. For deployments outside of this specific e2e demo, multi node clusters are definitely supported. Or something like that.

Reply:
@jiria how do the changes in 8ddc545ee7 look? I am open to adjusting again so it makes the most sense!

Reply:
Looks good, thanks Kate.